test/upgradeability/OwnedUpgradeabilityProxy.test.js

```diff
@@ -51,7 +51,7 @@ contract('OwnedUpgradeabilityProxy', ([_, owner, anotherAccount, implementation_
       })
     })
 
-    describe('when there is no implementation set', function () {
+    describe.skip('when there is no implementation set', function () {
```

Comment:
This test no longer applies. I didn't delete it because I wanted to discuss it further.

I don't think an `UpgradeabilityProxy` should be constructed without a valid implementation. I'd propose to change its constructor to take an initial implementation address. This affects the factory implementation because instead of deploying the proxy and then doing `upgradeToAndCall`, it will have to deploy it with its implementation set, and then call the function manually.